### PR TITLE
XTerm installation page when using a package manager

### PR DESCRIPTION
**Pull request by max-cx**
@ThomasDickey, if you can review this, it'd be great. Just one file.

Any comments from you are very welcome in the *Files changed* tab here.

Here is the preview to read: https://github.com/xterm-x11/docs/blob/5a049a0b9d39147f24fb0150c41892d3635aaaa1/package.md.

BTW, I commented out the command for SUSE because it didn't work for me and I'll have to look for external help, prbly the SUSE community, to check the installation command and options before I can publish it.

### CODE REVIEW

**Comment by ThomasDickey** [comment]
I also develop with OpenSUSE and Mageia (both rpm-based) and Slackware, though most of that consists of using the tools on the test-packages which I create ([zypper](https://en.opensuse.org/SDB:Zypper_usage) and [urpmi](https://wiki.mageia.org/en/URPMI) for the former, and [installpkg](https://docs.slackware.com/slackbook:package_management) for the latter).

**Comment by max-cx** [comment]
Alright. I'll have a second look.

BTW, I've drafted a separate page for the BSD distros, but it's more of a list of links to the xterm packages rather than the actual installation commands.

File: installing-xterm-from-a-package-on-gnulinux-distributions.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Fedora, Rocky Linux, CentOS Linux, CentOS Stream, RHEL
+
+The CLI command for installing XTerm from a package by using [DNF](https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/package-management/DNF/) on [Fedora](https://getfedora.org/), [Rocky Linux](https://rockylinux.org/), [CentOS Linux](https://www.centos.org/centos-linux/), [CentOS Stream](https://www.centos.org/centos-stream/), or [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux):
+
+`sudo dnf install xterm.x86_64`
+
+> For the XTerm package info, see Fedora Package Sources: https://src.fedoraproject.org/rpms/xterm.
+
+<!--
+## openSUSE/SUSE Linux Enterprise
```
Comment (max-cx) [inline installing-xterm-from-a-package-on-gnulinux-distributions.md:53]: I'm going to ask somebody to confirm the installation command in the https://forums.opensuse.org/forumdisplay.php/669-Applications forum.

Comment (max-cx) [reply installing-xterm-from-a-package-on-gnulinux-distributions.md:53]: @ThomasDickey, FYI:
https://forums.opensuse.org/showthread.php/572947-Docs-related-request-about-installing-XTerm-on-openSUSE-and-SUSE-Linux-Enterprise?p=3144613#post3144613

Comment (max-cx) [reply installing-xterm-from-a-package-on-gnulinux-distributions.md:53]: My next stops to ask:
https://forums.rancher.com/c/suse-product-topics/suse-package-hub/92
https://forums.rancher.com/c/suse-product-topics/suse-linux-enterprise-desktop/33
https://forums.rancher.com/c/suse-product-topics/suse-linux-enterprise-server/32

File: installing-xterm-from-a-package-on-gnulinux-distributions.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The command for installing XTerm from a package by using [Zypper](https://doc.opensuse.org/documentation/leap/reference/html/book-reference/cha-sw-cl.html#sec-zypper) on a command line on [openSUSE](https://www.opensuse.org/) or [SUSE Linux Enterprise products](https://www.suse.com/products/):
+
+`zypper install xterm`
```
Comment (max-cx) [inline installing-xterm-from-a-package-on-gnulinux-distributions.md:57]:
```suggestion
`sudo zypper install xterm`
```

File: installing-xterm-from-a-package-on-gnulinux-distributions.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Installing XTerm on GNU/Linux by using a package manager
+
+A long yet incomplete list of the XTerm packages across various GNU/Linux distributions is available at [pkgs.org](https://pkgs.org/download/xterm).
```
Comment (max-cx) [inline installing-xterm-from-a-package-on-gnulinux-distributions.md:3]: I'm thinking of moving this par to the bottom of this section, along with the links to XTerm packages on a few other distros that I discovered.

Comment (max-cx) [reply installing-xterm-from-a-package-on-gnulinux-distributions.md:3]: I need to add, after the pkgs.org link/list:
.
the package page for xterm on Gentoo Linux (https://www.gentoo.org/)
https://packages.gentoo.org/packages/x11-terms/xterm

Sabayon/Funtoo Linux https://www.sabayon.org/ and https://www.funtoo.org
the oswatershed.org info may be outdated, so ask on https://forum.sabayon.org if there is currently an xterm package

**Comment by max-cx** [comment]
@ThomasDickey, this PR is ready for your review.
 Here's the preview: https://github.com/xterm-x11/docs/blob/2d79b69bdea27a1ef61f3b293a5c0237041d084a/package.md

**Comment by ThomasDickey** [comment]
looks ok - you might not have noticed [repology](https://repology.org/project/xterm/versions), which has a list...

**Comment by max-cx** [comment]
Thank you, Thomas. I will look into incorporating the [repology](https://repology.org/project/xterm/versions) link into the text and adding any missing distributions to the list here.

**Comment by max-cx** [comment]
Just adding the missing `$` to some commands in the first section:
https://github.com/xterm-x11/docs/blob/074f272046773fb07ff34a4ff75c8c267bf16004/package.md

File: package.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+`$ loginctl show-session $(loginctl show-user $(whoami) -p Display --value) -p Type --value`
+
+`$ loginctl show-session "$XDG_SESSION_ID" -p Type`
+
+<!--
+Also $ echo $WAYLAND_DISPLAY (prints nothing if Wayland is not used)
+Source for thee above commands:
+https://unix.stackexchange.com/questions/202891/how-to-know-whether-wayland-or-x11-is-being-used/371164#371164
+-->
+
```
Comment (max-cx) [inline package.md:33]:
```suggestion

1. Some distributions like Debian or openSUSE have XTerm pre-installed. So you can check if XTerm is already installed on your system by running `$ xterm`. If it is indeed pre-installed, skip the next steps and go to the last step, which is post-installation configuration.
```

**Comment by max-cx** [comment]
Latest preview: https://github.com/xterm-x11/docs/blob/bce4c9e92c2e6d98747fe0a77daa0932afc17469/package.md

**Comment by ThomasDickey** [comment]
looks okay

**Pull request merged**